meeseeksdev/meeseeksbox/commands.py

```diff
@@ -310,6 +310,11 @@ def black_suggest(*, session, payload, arguments, local_config=None):
             except Exception:
                 # likely unprecessable entity out of range
                 pass
+    if os.path.exists(repo_name):
+        print("== Cleaning up repo... ")
+        subprocess.run("rm -rf {}".format(repo_name).split(" "))
```

Comment:
```suggestion
        subprocess.run(["rm", "-rf", repo_name])
```
is safer and less complicated (though I suspect `repo_name` will never have spaces.) But you may also prefer to use `shutil.rmtree`.

Reply:
Yeah, it was just pure lazyness and copy-past.